lib/logging.go

```diff
@@ -17,6 +19,22 @@ var (
 	TraceLogging         = slog.Levels{slog.PanicLevel, slog.FatalLevel, slog.ErrorLevel, slog.WarnLevel, slog.InfoLevel, slog.NoticeLevel, slog.DebugLevel, slog.TraceLevel}
 )
 
+func NewStderrConsoleWithLF(lf slog.LevelFormattable) *handler.ConsoleHandler {
+	h := handler.NewIOWriterWithLF(os.Stderr, lf)
+
+	// default use text formatter
+	f := slog.NewTextFormatter()
+	// default enable color on console
+	f.WithEnableColor(color.SupportColor())
```

Comment:
Not sure how this works as it still produces color output for me when redirecting to a file which in general should not happen with color output.
Btw we have a feature request to enable/disable color based on condition: https://github.com/warrensbox/terraform-switcher/issues/462 =)

Reply:
Were you hoping anything to be done in this PR or wait for #462  - as suggested, this is copied from upstream for now, to just change the output :)

Reply:
Nope =) PR looks (and works) good to me. Just need Warren to review and approve I guess.